src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -113,6 +117,10 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		this._register(this._configurationService.onDidChangeConfiguration(e => {
 			this.previewFeaturesEnabled = this._configurationService.getValue('workbench.enablePreviewFeatures');
 		}));
+		this.doubleClickEditEnabled = this._configurationService.getValue('notebook.enableDoubleClickEdit');
+		this._register(this._configurationService.onDidChangeConfiguration(e => {
```

Comment:
I don't think this listener is needed since each time this.doubleClickEditEnabled is used, it gets the value from configurationService again.

Reply:
I attempted to remove this, but it does not get the value from configservice every time, and it needs this listener.